tutorials/fundamentals/let-vs-var.md

```diff
@@ -0,0 +1,21 @@
+In JavaScript, the programmer has two options when declaring a mutable variable: `let` and `var`.
```

Comment:
"In JavaScript, you can use `let` or `var` to declare a mutable variable." Better to use fewer words, and use 2nd person rather than abstract 3rd person.